<?xml version="1.0"?>
<launch>

  <!-- <arg name="rgb_image_topic" default="/kinect2/qhd/image_color"/>
  <arg name="rgb_camera_info_topic" default="kinect2/qhd/camera_info"/> -->

  <arg name="global_frame_id" default="base_footprint"/>

  <arg name="marker_size" default="4.7" />
  <arg name="max_new_marker_error" default="0.08"/>
  <arg name="max_track_error" default="0.2"/>

  <arg name="publish_viz" default="true"/>
  <arg name="publish_tf" default="true"/>

  <node name="ar_perception" pkg="dt_uwds3_scripts" type="ar_perception_node.py" args="" output="screen" respawn="false">
    <rosparam subst_value="true">

      global_frame_id: $(arg global_frame_id)

      publish_viz: $(arg publish_viz)
      publish_tf: $(arg publish_tf)

    </rosparam>
  </node>

</launch>
